src/views-components/dialog-create/dialog-collection-create.tsx

```diff
@@ -73,7 +84,7 @@ export const DialogCollectionCreate = compose(
                     maxWidth='sm'
                     disableBackdropClick={true}
                     disableEscapeKeyDown={true}>
-                    <form onSubmit={handleSubmit((data: any) => onSubmit(data))}>
+                    <form onSubmit={handleSubmit((data: any) => onSubmit(data, files))}>
```

Comment:
Avoid creating functions in component props. In this case you could pass `onSubmit` to reduxForm config and call props.handleSubmit on button click.